vllm/v1/core/scheduler.py

```diff
@@ -227,13 +228,12 @@ def update_from_output(
         self,
         scheduler_output: "SchedulerOutput",
         model_runner_output: "ModelRunnerOutput",
-    ) -> List[Tuple[Request, int]]:
+    ) -> List[EngineCoreOutput]:
```

Comment:
Im not sure it makes sense for this method to be in `scheduler.py`

The only item related to setting the scheduler here is updating which `self.running`